pkg/virt-launcher/virtwrap/live-migration-source.go

```diff
@@ -360,7 +360,7 @@ func (l *LibvirtDomainManager) initializeMigrationMetadata(vmi *v1.VirtualMachin
 
 func (l *LibvirtDomainManager) cancelMigration(vmi *v1.VirtualMachineInstance) error {
 	migration, _ := l.metadataCache.Migration.Load()
-	if migration.Completed || migration.Failed || migration.StartTimestamp == nil {
+	if migration.Failed || migration.StartTimestamp == nil {
```

Comment:
Don't you want to check for end timestamp?

Reply:
yes you're right.